korman/properties/modifiers/avatar.py

```diff
@@ -77,6 +77,8 @@ def export(self, exporter, bo, so):
         simIface, physical = exporter.physics.generate_physical(bo, so, bounds, det_name)
         physical.memberGroup = plSimDefs.kGroupDetector
         physical.reportGroup |= 1 << plSimDefs.kGroupAvatar
+        physical.setProperty(plSimulationInterface.kPinned, True)
```

Comment:
This needs to be set on the `plSimulationInterface` as well. It's functionally a no-op, but I prefer for the bit flags to be identical on the two objects.

Reply:
Done.